source/Halibut.Tests/Builders/ServiceFactoryBuilder.cs

```diff
@@ -27,8 +29,9 @@ public ServiceFactoryBuilder WithService<TContract>(Func<TContract> factoryFunc)
             }
             // Convention verification may throw, but that just means we're probably going to use
             // the other factory anyway, so we don't care!
-            catch
+            catch (Exception e)
             {
+                conventionExceptions.Add(e);
```

Comment:
I needed to see what was going wrong when I incorrectly converted a service to async.